<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>400</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item row="15" column="1">
      <widget class="QPushButton" name="start_btn">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QProgressBar" name="progress_bar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="rewriting_files_chk">
       <property name="text">
        <string>Override files with same name</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="timer_period_lbl">
       <property name="text">
        <string>Timer period in secs</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="mask_ledit">
       <property name="placeholderText">
        <string>use space separated qt wildcards </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="mask_btn">
       <property name="text">
        <string>Show found files</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="value_lbl">
       <property name="text">
        <string>Transforming value</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QRadioButton" name="value_directly_rbtn">
       <property name="text">
        <string>Set bytes by hex</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="choose_folder_btn">
       <property name="text">
        <string>Choose folder</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="status_lbl">
       <property name="text">
        <string>Status: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="input_deleting_chk">
       <property name="text">
        <string>Delete input files</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mask_lbl">
       <property name="text">
        <string>File mask</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="saving_folder_ledit"/>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="use_timer_chk">
       <property name="text">
        <string>Use timer</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="status_value_lbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QTimeEdit" name="timer_period_tedit">
       <property name="minimumTime">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>1</second>
        </time>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::Section::SecondSection</enum>
       </property>
       <property name="displayFormat">
        <string>s</string>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>1</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QRadioButton" name="value_nums_rbtn">
       <property name="text">
        <string>Set bytes by numbers</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="saving_folder_lbl">
       <property name="text">
        <string>File saving folder</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QRadioButton" name="value_text_rbtn">
       <property name="text">
        <string>Set bytes by text</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="value_ledit"/>
     </item>
     <item row="16" column="1">
      <widget class="QPushButton" name="timer_stop_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop timer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
